lib/ssh/scp.go

```diff
@@ -51,7 +51,8 @@ func PutFile(ctx context.Context, client *ssh.Client, log logrus.FieldLogger, sr
 	}
 
 	go func() {
-		err := scpSendFile(stdin, f, fi)
+		buf := make([]byte, 32<<20)
```

Comment:
By default, `io.Copy` uses 32KiB intermediate buffer. This is a yet unjustified attempt to speed up (installer) file transfer.

Reply:
32<<20 is a bit of a hard read for me...

why not `1<<25`
or
```
const MiB = 1048576 
buf := make([]byte, 32*MiB)
```
?